testing/environments/5.0.0-alpha4.yml

```diff
@@ -3,14 +3,14 @@
 
 elasticsearch:
   build: ./docker/elasticsearch
-  dockerfile: Dockerfile-5.0.0-alpha4
+  dockerfile: Dockerfile-5.0.0-alpha5
   command: elasticsearch -Enetwork.host=0.0.0.0 -Ediscovery.zen.minimum_master_nodes=1 -Ebootstrap.ignore_system_bootstrap_checks=true
 
 logstash:
   build: ./docker/logstash
-  dockerfile: Dockerfile-5.0.0-alpha4
+  dockerfile: Dockerfile-5.0.0-alpha5
 
 kibana:
   build: ./docker/kibana
-  dockerfile: Dockerfile-5.0.0-alpha4
+  dockerfile: Dockerfile-5.0.0-alpha5
```

Comment:
Oh, I just realised I didn't push my most recent change locally as I forgot to use force -> see also comment below